css/style.css

```diff
@@ -5,7 +5,8 @@ html {
 }
 
 img {
-  width: 100%;
+  width: 30%;
```

Comment:
Don't forget that this affects _all_ `img`s across the application... that's probably not what you want. `100%` makes sense as a default, but 30% probably does not. Same for the border.

How can you make these changes more _specific_ to that header image?